src/index.tsx

```diff
@@ -237,7 +237,24 @@ const ReactSearchInternal: FC<Props> = ({
         <SearchModal isOpen={isOpen} onClose={closeModalAndResetResults}>
           <form>
             <div className="vrsSearchForm">
-              <SearchInput value={searchValue} onChange={onChange} onKeyDown={onKeyDown} placeholder={placeholder} />
+              <div className="vrsCloseButtonWrapper">
+                <button
```

Comment:
Can we give this an [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) to describe it to screen readers?